doc/Installation.md

```diff
@@ -0,0 +1,152 @@
+# INSTALL #
+
+This document explains how easy is to get tfchain up and running.
+
+## Using dockerfile ##
+
+## Using docker ##
+
+In this instalation we assume that you are working on a linux machine with docker installed.
+
+You need to have an image for a linux distribution, from now on we will use Ubuntu 16.04 as the one, but you may chose one that best suits you as long as a go development environmet can run on it (all of the modern ones).
+
+First when in a graphical environment, we need to open a terminal (like xterminal, )
+
+
+So we start downloading our own ubuntu image
+```
+docker pull ubuntu:16.04
```

Comment:
most of the steps after this really are a bit over-the-top. We already provide tfchain containers. I think if people know how to build from source (which we do explain) and know how to use docker, they can figure out how to build and run from within docker as well (just building alone we provide by the way a container and script for as well, but that's out of scope for this doc as well).

For our docs concerned the only thing that should matter for them is that they can install with docker by simply pulling our image (`tfchain:v0.1.0`), and that's it. From than on they can run the docker container image, very similar to how they would use a local binary. The docker-specific details should be clear to any docker user, as those are not specific to tfchain. You could always link to the latest docker docs to learn more about that, for those that are intrigued by docker container versions, but don't know docker (yet).

Reply:
the missing part about dockerfile is just that. a quick up and running installation without hassle.

But giving the detailed instructions does not bother as other persons can be interested in additional detail.